<launch>
  <rosparam command="load" file="$(find sub_trajectory)/param/high_level_thrusters.param"/>
  <node name="vector_control" pkg="sub_trajectory" type="vector_thrust_control.py" output="screen">
  
  </node>
</launch>
